src/interfaces/IStrategy.sol

```diff
@@ -27,4 +19,12 @@ interface IStrategy {
 
     /// @notice Recovers ERC20 tokens from the strategy
     function recoverERC20(address _token, address _recipient, uint256 _amount) external;
+
+    /// @notice Requests a withdrawal from the Withdrawal Queue
+    function requestWithdrawalFromPool(
+        uint256 _stvToWithdraw,
+        uint256 _stethSharesToBurn,
+        uint256 _stethSharesToRebalance,
+        address _receiver
+    ) external returns (uint256 requestId);
```

Comment:
🤔 now burning and rebalancing are separated in the Pool contract. Should we do the same here? I think this could help us work more flexibly with the strategy

I mean having two separate methods: burnWsteth, requestWithdrawal (with rebalancing but without burning)